EIPS/eip-867.md

```diff
@@ -0,0 +1,195 @@
+
+Once an ERP is published (via a pull request made to the EIP repository), the approval process begins.  The author is encouraged to seek feedback from the following groups:
+
+- People or organizations who are directly affected by the issue (e.g. entities who will be refunded or were associated with the original incident)
+- People or organizations that are unaffected by the issue but may serve as relatively impartial reviewers and have the technical ability to confirm or reject the factual claims made in the ERP.
+
+ERPs that have strong support from both groups of people will be much more likely to succeed.  The first group, although not impartial, is critical to determine that among the directly affected parties there is broad agreement about the desired outcome.  The second group is critical to help verify (a) the conditions defined in the verification script accurately represent the human readable version, (b) that the verification script generates the State Change Object submitted with the ERP, and (c) that the people/orgs representing the “affected parties” is representative.  
+
+Positive, explicit, and broad support from both groups will be critical to the approval process; a lack of negative feedback is not sufficient.  It is the sole responsibility of the ERP author and affected parties to find support. Feedback that is not substantive should be dismissed, and feedback for the ERP process as a whole should be directed to this meta-EIP.  Community members, in general, are encouraged to reference these guidelines to ensure the process remains focused, objective, and expedient.  
+
+The ERP author may indicate in the ERP comments when he/she believes there is sufficient support and request that the ERP be considered by client teams for inclusion in some future fork.  Client teams may have their own procedure for evaluating the ERP's merit, although they are encouraged to make the process as transparent as possible.
+
+Once an ERP meets the minimum requirements and client teams have indicated their intention to include the ERP as part of some future upgrade, the block at which the changes will be applied will be finalized and the ERP should be marked as "Accepted".  In the event of a disagreement between client teams, supporting client teams may choose to delay the ERP's inclusion to avoid a split chain.  Client teams that are not supportive are encouraged to give feedback to the ERP authors so that they might attempt to address their concerns.
+
+The target block will be determined based on coordination between the client teams and other pending releases.  Upon determination of the target block, a finalized version (updated to include the determined block) of the State Change Object will be published to the ERP.  Clients that support the inclusion of the ERP should bundle this final version (unmodified) with their respective applications.    
+
+### Testing
```

Comment:
Should be a `##` second level headline below _Implementation_.

Reply:
There are two types of testing here.  One is testing that the new client code (which would be written once and handle all ERPs).  The other is testing a particular ERP to ensure the state changes do what was intended.  This section was intended to be about the ERP state change testing, so I think should go under the approval process.

Reply:
Got it.